src/rebar3_hex_key.erl

```diff
@@ -1,141 +1,154 @@
 -module(rebar3_hex_key).
 
--export([init/1,
-         do/1,
-         format_error/1]).
+-export([init/1, do/1, format_error/1]).
 
 -include("rebar3_hex.hrl").
 
 -define(PROVIDER, key).
 -define(DEPS, []).
 
--define(ENDPOINT, "keys").
-
 -spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
 init(State) ->
     Provider = providers:create([{name, ?PROVIDER},
                                  {module, ?MODULE},
                                  {namespace, hex},
                                  {bare, true},
                                  {deps, ?DEPS},
-                                 {example, "rebar3 hex key [generate <key> | list | revoke <key> | revoke --all]"},
-                                 {short_desc, "Remove or list API keys associated with your account"},
+                                 {example,
+                                  "rebar3 hex key [generate -k <key> | list | revoke -k <key> "
```

Comment:
this should probably come with a major version bump? Or is this considered a bug fix?

Reply:
Two bug fixes... one in the handling of the args and command, second in the documentation.